.github/workflows/cache_server.test.yml

```diff
@@ -0,0 +1,66 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events
+  push:
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+
+  deployment:
+    services:
+      postgres:
+       image: postgres
+       env:
+        DB_USERNAME: ${{secrets.DB_USERNAME}}
+        #DB_USERNAME: 'postgres'
+        DB_PASSWORD: ${{secrets.DB_PASSWORD}}
+        #DB_PASSWORD: 'password'
+        DB_HOST: 'postgres'  
+       ports:
+       - 5432:5432
+      redis:
+        image: redis
+        env:
+            REDIS_PASSWORD: ${{secrets.REDIS_PASSWORD}}
+            #REDIS_PASSWORD: 'passord'
+            REDIS_HOST: 'redis'
+        ports:
+        - 6379:6379
+      
+      # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      # Runs a single command using the runners shell
+      - name: Install dependencies
+        run: npm install
+        
+      - name: Docker Build
+        run: npm run docker:build -f "docker-compose.dev.yml"
```

Comment:
I'm a bit confused. We're running the whole dev `docker-compose` but we're also running services? I guess what would make sense to me is if we run the `docker-compose.prod.yml` (as that is what we are running in production) and then only run `postgres` as a service in the GHA. How does that sound?

Reply:
Yes that sounds indeed balanced. Is it ok if I provide a docker-compose.ci.yml ? Since I didn't use required values like AWS ECR credentials.